### ISSUE TITLE
Creating 2 cohorts for most templates

### ISSUE THREAD

**Issue by cborn** [catalyst/moodle-local_cohortauto#4]
Most, if not all, of our auto-generated cohorts are duplicated with some but not complete overlap in membership.

In some cases, I could see it if there is a multi-value field.  But I'm not sure that is the case.  What are the reasons cohorts would be duplicated?

**Comment by DrCuriosity** [comment]
Apologies for the late reply - things have been... frenetic, recently.

I'm afraid I haven't been able to replicate that issue. When you are getting duplicated cohorts, do they both have the same "Managed by" field in `convert.php`?

**Comment by cborn** [comment]
Yes, they do.  I haven't confirmed a pattern yet, but I am wondering if it has to do with fields that have multiple values.  But, if that was related, why would there not be the same number of cohorts as there are possible values in these fields?

**Comment by cborn** [comment]
Here's an example, this is a small department.  All of three of these people have "Academic Support Center" as their primary department.  But they are spread across 2 groups.
![Screen Shot 2020-06-08 at 12 06 52 PM](https://user-images.githubusercontent.com/1149264/84060694-4368b680-a982-11ea-8dd6-87895a5a63c8.png)

**Comment by danmarsden** [comment]
It requires an exact match - I'd guess that your data comes from somewhere where a user enters them manually and has added a space to the end or even 2 spaces between the words.

**Comment by cborn** [comment]
@danmarsden Good idea, and so I checked the data. Sadly, that did not show true in the few that I checked.  And it's far too systematic to suggest poor data entry.  I have 2 cohorts for each and every department at our college, some of them are even empty.
